index.html

```diff
@@ -474,6 +516,12 @@ <h3>Programming Diversity</h3>
                 };
 				 return t;
             }(document, "script", "twitter-wjs"));
+
```

Comment:
is there a reasoning behind this amount of blank spaces? Perhaps left in while developing?